### PR TITLE
convert external blog posts

### PR DESCRIPTION
**Pull request by reddypramod85**
This PR converts the external blog posts from the existing HPE DEV CMS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # 🚁 < 🛸
+
+cd post-conveter-script
```
Comment (MikeKingdom) [inline README.md:3]: typo: post-conveter-script -> post-converter-script

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+npm install or yarn install
```
Comment (MikeKingdom) [inline README.md:4]: really should stick with one or the other (preferably yarn)

File: content/2017-09-06-authenticating-against-the-composable-api.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Authenticating against the Composable API
+title: "Blog"
```
Comment (MikeKingdom) [inline content/2017-09-06-authenticating-against-the-composable-api.md:2]: this looks wrong

**Review by MikeKingdom — COMMENTED** [review]
Why commit all the converted files to here?

File: content/2017-09-06-authenticating-against-the-composable-api.md (hunk 3 above)
Comment (reddypramod85) [reply content/2017-09-06-authenticating-against-the-composable-api.md]: Yes, I realized it after pushing and it's fixed now.

File: README.md (hunk 2 above)
Comment (reddypramod85) [reply README.md:4]: Okay, I'll stick to yarn.

File: README.md (hunk 1 above)
Comment (reddypramod85) [reply README.md]: Good catch, it's fixed.

**Comment by reddypramod85** [comment]
> Why commit all the converted files to here?

Initially, the content folder was not ignored, so all the converted files were checked in. It makes sense to ignore this folder and added it to .gitignore.

**Pull request merged**